### DISCUSSION TITLE
Windows 8.1

### DISCUSSION THREAD

**Discussion by re-vliz** [pear-devs/pear-desktop#1503] [Q&A] [answered]
Not working in windows 8.1 x64?
Which version still supports Windows 8.1?

**Comment by ArjixWasTaken** [comment, marked as answer]
https://www.electronjs.org/blog/windows-7-to-8-1-deprecation-notice

[Here](https://github.com/th-ch/youtube-music/releases/tag/v1.20.0
) is the latest version that is using electron-22 which still supports Windows 8.

That is a pretty old version though, and Windows 8.1 released in 2013...
I'd really suggest you update both, you are probably missing important security updates and may be vulnerable to hackers.
